src/test/TemplateProcessor.test.js

```diff
@@ -2109,9 +2108,13 @@ test("don't re-evaluate intervals", async () => {
         "stop": "${ count=10?($clearInterval($$.counter);'done'):'not done'  }"
     }
     const tp = new TemplateProcessor(template);
-    await tp.initialize();
-    const from = tp.from("/count");
-    expect(from).toEqual(["/count", "/stop"]);
+    try {
+        await tp.initialize();
+        const from = tp.from("/count");
+        expect(from).toEqual(["/count", "/stop"]);
+    } finally {
+        await tp.close();
```

Comment:
should we make sure all our examples and tests close() the tp? (good catch in this one, as it has interval/timeout. Probably we need to close() as a best practice, everywhere?